lib/jsdom/living/nodes/HTMLTableCellElement-impl.js

```diff
@@ -3,29 +3,24 @@
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
 
 const closest = require("../helpers/traversal").closest;
-const createDOMTokenList = require("../dom-token-list").create;
-const resetDOMTokenList = require("../dom-token-list").reset;
 
 class HTMLTableCellImpl extends HTMLElementImpl {
-  get headers() {
-    if (this._headers === undefined) {
-      this._headers = createDOMTokenList(this, "headers");
-    }
-    return this._headers;
-  }
-
-  get cellIndex() {
-    const tr = closest(this, "tr");
-    if (tr === null) {
-      return -1;
-    }
-
-    return tr.cells.indexOf(this);
-  }
-
   get colSpan() {
     const value = this.getAttribute("colspan");
-    return value === null ? 1 : value;
+    if (value === null) {
```

Comment:
Given the duplicate colSpan/rowSpan code, we should factor this out, so that it's something like `return reflectedAttributeWithinRange(this.getAttribute("colspan"), 1, 1000)`